content/en/docs/contribute/generate-ref-docs/metrics-reference.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Generating reference documentation for metrics

```

Comment:
Thanks

Please also add a suitable `weight` and a `content_type` (which should be `task`)